modules/julia/CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+    return()
+else()
+    message(STATUS "JlCxx_DIR: ${JlCxx_DIR}")
+endif()
+
+set(the_description "The Julia bindings")
+ocv_add_module(julia    BINDINGS
+                                 opencv_core
+                                 opencv_imgproc 
+                                 opencv_imgcodecs
+                                 opencv_objdetect
+                                 opencv_highgui
+                                 opencv_features2d
+                                 opencv_videoio
+)
+
+ocv_glob_module_sources()
+ocv_module_include_directories()
+
+ocv_add_library(${the_module} SHARED ${OPENCV_MODULE_${the_module}_HEADERS} ${OPENCV_MODULE_${the_module}_SOURCES})
+
+if(NOT JULIA_PKG_INSTALL_PATH)
+    set(JULIA_PKG_INSTALL_PATH ${CMAKE_CURRENT_BINARY_DIR})
+endif()
+message(STATUS "Installing ${the_module} bindings at ${JULIA_PKG_INSTALL_PATH}")
+
+
+install(DIRECTORY package/ DESTINATION ${JULIA_PKG_INSTALL_PATH})
+install(TARGETS ${the_module} LIBRARY DESTINATION ${JULIA_PKG_INSTALL_PATH}/OpenCV/src/lib)
+
+
+if(JULIA_PKG_INSTALL_ENV)
+    set(JULIA_PKG_EXECS "Pkg.activate(${JULIA_PKG_INSTALL_ENV});${JULIA_PKG_EXECS}")
+endif()
+
+
+set(JULIA_COMMAND "\"${Julia_EXECUTABLE}\" -e \"using Pkg; ${JULIA_PKG_EXECS}\"")
+message(STATUS "Installing Julia bindings using ${JULIA_COMMAND}")
+
+
+install(CODE "
+set(JULIA_PKG_PATH \"${JULIA_PKG_INSTALL_PATH}/OpenCV\")
+execute_process(COMMAND \"${Julia_EXECUTABLE}\" ${CMAKE_CURRENT_LIST_DIR}/package/install_package.jl \${JULIA_PKG_PATH} OUTPUT_VARIABLE JULIA_INSTALL_OUT)
+
+
+# set(JULIA_PKG_EXECS \"Pkg.develop(PackageSpec(path=\\\\\\\"${JULIA_PKG_INSTALL_PATH}/OpenCV\\\\\\\"));\")
+
+# if(${JULIA_PKG_INSTALL_ENV})
+#     set(JULIA_PKG_EXECS \"Pkg.activate(${JULIA_PKG_INSTALL_ENV});\${JULIA_PKG_EXECS}\")
+# endif()
+# set(JULIA_COMMAND \"\\\"using Pkg; \${JULIA_PKG_EXECS}\\\"\")
+
+# message(STATUS \"Installing Julia bindings using \${JULIA_COMMAND}\")
+
+# execute_process(COMMAND \"${Julia_EXECUTABLE}\" -e \${JULIA_COMMAND} OUTPUT_VARIABLE JULIA_INSTALL_OUT)
+
+message(STATUS \"Install output: \${JULIA_INSTALL_OUT}\")
+")
+
+# set(JULIA_COMMAND \"\"${Julia_EXECUTABLE}\" -e \"using Pkg; \${JULIA_PKG_EXECS}\"\")
+
+# ocv_create_module()
+
+ocv_add_accuracy_tests()
+ocv_add_perf_tests()
+ocv_add_samples()
+
+message(STATUS     ${OPENCV_MODULE_${the_module}_DEPS_TO_LINK})
+
+ocv_target_link_libraries(${the_module} PUBLIC    ${OPENCV_MODULE_${the_module}_DEPS_TO_LINK}
+                                        INTERFACE ${OPENCV_MODULE_${the_module}_DEPS_TO_LINK}
+)
+ocv_target_link_libraries(${the_module} PUBLIC    ${OPENCV_MODULE_${the_module}_DEPS_EXT}
+                                        INTERFACE ${OPENCV_MODULE_${the_module}_DEPS_EXT}
+)
+ocv_target_link_libraries(${the_module} PRIVATE ${OPENCV_LINKER_LIBS} ${OPENCV_HAL_LINKER_LIBS} ${IPP_LIBS} ${ARGN})
+
+ocv_target_link_libraries(${the_module} JlCxx::cxxwrap_julia)
+ocv_target_link_libraries(${the_module} JlCxx::cxxwrap_julia_stl)
```

Comment:
I do not see where "OpenCV.jl" is generated

Reply:
It's done in the `INSTALL(CODE ...)` part